### ISSUE TITLE
getting jibMaven dependencies: unable to stat file C:\\demos\\my-demo\\pom.xml\r:

### ISSUE THREAD

**Issue by dkirrane** [GoogleContainerTools/skaffold#1159]
Trying out the skaffold-jib support but hitting the following:
```
$ skaffold dev
Cleaning up...
Error: release: "my-demo" not found
Cleanup complete in 4.9536661s
time="2018-10-15T10:51:34+01:00" level=fatal msg="watching files for artifact nexus-mycompany.com/my-demo: listing files: listing files: getting jibMaven dependencies: unable to stat file C:\\demos\\my-demo\\pom.xml\r: CreateFile C:\\demos\\my-demo\\pom.xml\r: The filename, directory name, or volume label syntax is incorrect."
```

### Expected behavior
Build, deploy and 

### Information

- Skaffold version: version: `v0.16.0`
- Operating system: `Windows 10` running from latest GitBash terminal
- Contents of skaffold.yaml:

```yaml
apiVersion: skaffold/v1alpha4
kind: Config

build:
  artifacts:
  - image: nexus-mycompany.com/my-demo
    jibMaven: 
      profile: prod
  tagPolicy:
      gitCommit: {}

deploy:
  helm:
    releases:
    - name: my-demo
      chartPath: helm
      valuesFiles:
        - values.yaml
      wait: true
      namespace: default
      version: "1.0"
```

### Steps to reproduce the behavior

1. create simple spring boot 2 app (version `2.0.5.RELEASE`)
2. added jib maven plugin to `pom.xml`
```
            <plugin>
                <groupId>com.google.cloud.tools</groupId>
                <artifactId>jib-maven-plugin</artifactId>
                <version>0.9.11</version>
            </plugin>
```
3. `skaffold dev`

**Comment by dgageot** [comment]
ping @GoogleContainerTools/java-tools

**Comment by briandealwis** [comment]
`C:\\demos\\my-demo\\pom.xml\r` &larr; looks like a carriage return has been appended!

**Comment by coollog** [comment]
Ah yes, since we only [split the Maven dep listing on `\n`](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/jib/jib.go#L39), on Windows, the Java line endings may be `\r\n`. Options include:

- Having windows users set line ending to `\n` via for e.g., `-Dline.separator=$'\n'`
- Changing out dep list processing to first replace all `\r\n` with `\n`

**Comment by loosebazooka** [comment]
Shouldn't golang's readline handle this correctly?

**Comment by briandealwis** [comment]
I'll take this — I have to fix #1152 for the same thing.
